algorithms/gr-dij/java/Dijkstra.java

```diff
@@ -0,0 +1,89 @@
+import java.lang.*;
+
+class Dijkstra {
+    // A utility function to find the vertex with minimum distance value,
+    // from the set of vertices not yet included in shortest path tree
+    static final int V = 9;
+
+    int minDistance(int dist[], Boolean sptSet[]) {
+        // Initialize min value
+        int min = Integer.MAX_VALUE, min_index = -1;
+
+        for (int v = 0; v < V; v++)
+            if (sptSet[v] == false && dist[v] <= min) {
```

Comment:
Codacy found an issue: [Avoid unnecessary comparisons in boolean expressions](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6333211)